### PR TITLE
WORKSPACE: fix deps to bazel 1.2.1

### PR DESCRIPTION
**Pull request by tsvisabo**
- fix dependencies to match bazel 1.2.1

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -6,96 +6,93 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 # EXTERNAL RULES
 http_archive(
     name = "com_github_grpc_grpc",
-    strip_prefix = "grpc-1.22.0",
+    sha256 = "d6af0859d3ae4693b1955e972aa2e590d6f4d44baaa82651467c6beea453e30e",
+    strip_prefix = "grpc-1.26.0-pre1",
     urls = [
-        "https://github.com/grpc/grpc/archive/v1.22.0.tar.gz",
+        "https://github.com/grpc/grpc/archive/v1.26.0-pre1.tar.gz",
```
Comment (thibault-martinez) [inline WORKSPACE]: `pre` ? Does that mean it's an alpha or something ?  Do we need this specific release ?

File: hub/iota/tests/local_pow.cc [hunk 2]
```diff
@@ -107,9 +107,10 @@ TEST_F(LocalPOWTests, CorrectDigest) {
   std::string prev = digest;
   for (uint32_t i = 1; i < powedTxs.size(); ++i) {
     free(digest);
-  /* EXPECT_EQ(powedTxs[i].substr(hub::iota::LocalPOW::BRANCH_OFFSET, 81),
-              resp.trunkTransaction);*/
-  /*  EXPECT_EQ(powedTxs[i].substr(hub::iota::LocalPOW::TRUNK_OFFSET, 81), prev);*/
+    /* EXPECT_EQ(powedTxs[i].substr(hub::iota::LocalPOW::BRANCH_OFFSET, 81),
+                resp.trunkTransaction);*/
+    /*  EXPECT_EQ(powedTxs[i].substr(hub::iota::LocalPOW::TRUNK_OFFSET, 81),
+     * prev);*/
```
Comment (thibault-martinez) [inline hub/iota/tests/local_pow.cc]: Why are they commented btw ?

File: WORKSPACE (hunk 1 above)
Comment (tsvisabo) [reply WORKSPACE]: we do, this only builds with bazel 1.2.1

File: hub/iota/tests/local_pow.cc (hunk 2 above)
Comment (tsvisabo) [reply hub/iota/tests/local_pow.cc]: correct, they should be removed since they are no longer relevant because bundle is built with entangled func

**Pull request merged**